src/datasources/cache/cache.first.data.source.ts

```diff
@@ -56,4 +56,26 @@ export class CacheFirstDataSource {
     await this.cacheService.set(key, field, rawJson, expireTimeSeconds);
     return data;
   }
+
+  /**
+   * Executes a POST request against an {@link url}, by using
+   * the injected NetworkService.
+   * Cache is not used by this function.
```

Comment:
I think this can be quite confusing – from 2 of the methods exposed by the `CacheFirstDataSource` one of the doesn't "cache".

I would say that it is better for consumers of this API to use the network data source directly as it is more explicit.

Reply:
I had the exact same thought when I wrote this part 🙂. I forgot to comment that in the PR but I was doubting we could just use `NetworkService` in the caller as you said, or create an `IDataSource` and make `CacheFirstDataSource` (with cache) and a new `DataSource` (without cache) implement it, or even have `getWithCache/postWithCache` counterparts to `get/post` functions.

Probably just using `NetworkService` is the simpler one, I'll make that change 🙂

Reply:
Changed in [1d66427](https://github.com/5afe/safe-client-gateway-nest/pull/132/commits/1d6642705e425416d8d1842dc5d02f6792202098)